test/model/content_test.dart

```diff
@@ -590,11 +596,12 @@ void main() {
     '<p><code>inline code</code></p>',
     const InlineCodeNode(nodes: [TextNode('inline code')]));
 
-  testParseInline('parse nested strong, em, code',
+  testParseInline('parse nested del, strong, em, code',
     // "***`word`***"
-    '<p><strong><em><code>word</code></em></strong></p>',
-    const StrongNode(nodes: [EmphasisNode(nodes: [InlineCodeNode(nodes: [
-      TextNode('word')])])]));
+    '<p><del><strong><em><code>word</code></em></strong></del></p>',
+    const DeletedNode(
+      nodes: [StrongNode(nodes: [EmphasisNode(nodes: [InlineCodeNode(nodes: [
+        TextNode('word')])])])]));
```

Comment:
The code-commented Markdown—

```dart
// "***`word`***"
```

should also be updated. (Here and in multiple places below.)

Reply:
That's right!

Reply:
@chrisbobbe Thanks for the feedback☺️
I've opened this before getting reviews on the previous PRs by Greg. 
I've learned a lot of things that can refactor this already! I will take a second go at it then mark it for review